gritulator/control/grid_forming/_power_synchronization.py

```diff
@@ -76,36 +60,52 @@ class PSCtrlPars:
 
 
 # %%
-class PSCtrl(Ctrl):
+class PSCCtrl(Ctrl):
     """
-    PSC control for grid converters.
+    power synchronization control for grid converters.
+    
+    This implements the power synchronization control (PSC) method described in
+    [#Har2019]_. The alternative reference-feedforward PSC (RFPSC) can also be 
+    used and is based on [#Har2020]_.
 
     Parameters
     ----------
     pars : PSCtrlPars
         Control parameters.
 
+    References
+    ----------
+    .. [#Har2019] Harnefors, Hinkkanen, Riaz, Rahman, Zhang, "Robust Analytic
+        Design of Power-Synchronization Control," IEEE Trans. Ind. Electron., Aug.
+        2019, https://doi.org/10.1109/TIE.2018.2874584
+        
+    .. [#Har2020] Harnefors, Rahman, Hinkkanen, Routimo, "Reference-Feedforward
+        Power-Synchronization Control," IEEE Trans. Power Electron., Sep. 2020,
+        https://doi.org/10.1109/TPEL.2020.2970991
+
     """
 
     # pylint: disable=too-many-instance-attributes, too-few-public-methods
     def __init__(self, pars):
         super().__init__()
-        self.t = 0
         self.T_s = pars.T_s
         # Instantiate classes
         self.pwm = PWM(pars)
+        self.clock = Clock()
         self.power_calc = PowerCalc(pars)
         self.current_ctrl = CurrentCtrl(pars)
         self.power_synch = PowerSynch(pars)
-        self.dc_voltage_control = DCVoltageControl(pars)
+        self.dc_bus_control = DCBusCtrl(pars.zeta_dc, pars.w_0_dc, pars.p_max)
```

Comment:
Could DCBusVoltCtrl name work for the class and similarly for self.dc_bus_volt_ctrl ? It would then clearly tell the signal that is controlled and where.

Reply:
Fixed.